tests/python/unittest/test_tvmscript_syntax_sugar.py

```diff
@@ -146,5 +146,16 @@ def test_match_buffer_syntax_sugar():
     assert_structural_equal(elementwise_handle, elementwise_buffer_no_kwargs)
 
 
+# match buffer failed case
+def test_match_buffer_no_kwargs_failed():
+    with pytest.raises(ValueError) as e:
+
+        def elementwise_buffer_no_kwargs_failed(
```

Comment:
```suggestion
        @T.prim_func
        def elementwise_buffer_no_kwargs_failed(
```
we'd like to make the test case as close to real use case as possible